src/elements/content-sidebar/activity-feed/app-activity/AppActivity.scss

```diff
@@ -0,0 +1,64 @@
+@import '../../../common/variables';
+
+.be {
+    .bcs-app-activity-container {
+        position: relative;
+        z-index: 1;
+
+        &:hover,
+        &.bcs-is-focused {
+            .bcs-inline-delete-container {
+                opacity: 1;
+            }
+        }
+
+        .bcs-app-activity {
+            display: flex;
+            margin-top: 23px;
+            opacity: .9999;
+            width: 100%;
+
+            &.bcs-is-pending {
+                opacity: .4;
+            }
+
+            .bcs-app-activity-icon {
+                flex-shrink: 0;
+                height: 32px;
+                margin-right: 10px;
+                width: 32px;
+            }
+
+            .bcs-app-activity-content {
+                color: $fours;
+                flex: 1;
+                min-width: 0;
+
+                .bcs-app-activity-link {
+                    color: $blue;
+                }
+
+                .bcs-app-activity-created-at {
+                    color: $sevens;
+                    cursor: default;
+                    font-size: 12px;
+                    line-height: 20px;
+                }
+            }
+
+            .bcs-app-activity-headline {
+                display: flex;
+                flex-flow: row wrap;
```

Comment:
I think the default value for `flex-direction` is `row`. Should this be `flex-wrap: wrap`?